src/Meshes/geometry.ts

```diff
@@ -584,12 +584,7 @@ export class Geometry implements IGetSetVerticesData {
         if (!forceCopy && (!copyWhenShared || this._meshes.length === 1)) {
             return orig;
         } else {
-            var len = orig.length;
-            var copy = [];
-            for (var i = 0; i < len; i++) {
-                copy.push(orig[i]);
-            }
-            return copy;
+            return orig.slice();

```

Comment:
Should we use `Tools.Slice` instead?

Reply:
It looks to me like `Tools.Slice` is useful if you don't know whether the object is an array, but in this case we do know it is an array. Is there another benefit?

Reply:
Looking at the doc, it says it is for IE support...

Reply:
Note to self: read doc comments.